layouts/shortcodes/vote.html

```diff
@@ -43,4 +44,12 @@ <h1 style="z-index: 999;position:relative">Earn tokens by voting here</h1>
             <span class="bubble"></span>
         </span></button></a>
     </div>
-</div>
\ No newline at end of file
+</div>
+<script type="text/javascript">
+    if (navigator.userAgent.indexOf('Safari') != -1  && navigator.userAgent.indexOf('Chrome') == -1) {
```

Comment:
You can wrap and invert this if statement to avoid having an if else

Reply:
I did it that way because I'm looking to eventually try to figure out how to add a custom method for Safari that fixes the UI difference. The code is meant to target Safari so I did it that way. I'll gladly inverse it if you'd still like though.

Reply:
Changes have been made.